### PR TITLE
Project generator

### PR DESCRIPTION
**Pull request by grabbou**
This pull request adds a new project generator using React Native templates feature.

Note that this feature is quite experimental and produces a lot of unnecessary files. That is the reason why I have included a Readme document in the generated project to help users understand the files present better.

### CODE REVIEW

File: docs/get-started.md [hunk 1]
```diff
@@ -6,7 +6,19 @@ sidebar_label: Quick Start
...
+
+You can create a new React Native project by running the following:
+
+```sh
+react-native init MyApp --template rebolt
+```
+
+This will create a fresh React Native project using `rebolt` template, giving you the already preconfigured Reason environment out of the box.
+
+To better understand the integration between Reason and React Native,please see the following section that explains the manual integration step by step.
```
Comment (knowbody) [inline docs/get-started.md]: nit: space missing
`Native,please`

File: generator/package.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "name": "react-native-template-rebolt",
+  "version": "0.0.3"
```
Comment (knowbody) [inline generator/package.json]: let's make it `0.0.1`

Comment (grabbou) [reply generator/package.json]: Haha, 0.0.1 is already on `npm` as I was testing it out xd

File: docs/get-started.md (hunk 1 above)
Comment (grabbou) [reply docs/get-started.md]: Yeah, fixed in the following commit

File: generator/package.json (hunk 2 above)
Comment (knowbody) [reply generator/package.json]: oh ok

**Comment by knowbody** [comment]
Thanks! This is great! 🎉

**Pull request merged**